src/plugins/data/public/antlr/opensearch_ppl/opensearch_ppl_autocomplete.ts

```diff
@@ -126,7 +145,7 @@ export function enrichAutocompleteResult(
   const result: OpenSearchPplAutocompleteResult = {
     ...baseResult,
     ...suggestionsFromRules,
-    suggestColumns: shouldSuggestColumns ? shouldSuggestColumns : undefined,
+    suggestColumns: shouldSuggestColumns ? ({ name: '' } as TableContextSuggestion) : undefined,
```

Comment:
`suggestColumns` expects `{ tables?: Table[]; }`, why is it `name` here?

Reply:
The Table interface contains a name field, but it seems like that isn't done right / didn't end up being needed anyways. I changed this to be an empty object casted to TableContextSuggestion